extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -491,9 +496,6 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
           keyName: TEMPORARY_STORE_EXTRA_ID,
         });
       } catch (e) {
-        captureException(
-          `Could not login when adding account: ${JSON.stringify(e)}`,
-        );
```

Comment:
removing this Sentry alert because this will be triggered every time a user enters an incorrect password. This will pollute our Sentry dashboard with non-errors